src/jshint.js

```diff
@@ -4225,7 +4225,7 @@ var JSHINT = (function() {
     var i = 0;
     var inof = ["in", "of"];
     var level = 0; // BindingPattern "level" --- level 0 === no BindingPattern
-    var comma; // First comma punctuator at level 0
+    var firstcomma; // First comma punctuator at level 0
```

Comment:
Hey sorry, 

We use this at http://www.bithound.io/ and I have seen it on a few private repos that we analyze.

We don't do anything magic, just call the JSHINT msg in our code:

```
if (!JSHINT(source + '\n', config, config.globals)) {
      JSHINT.errors.forEach(function (error) {
        if (error) {
          errors.push(error);
        }
      });
    }
}
```